bokchoi/gcp/gcp.py

```diff
@@ -262,7 +274,25 @@ def run(self):
         """Run the uploaded package"""
         create_instance_op = self.create_instance()
         self.wait_for_operation(create_instance_op)
-        return 'Running application'
+        print('Running application')
+        if self.wait_for_execution:
```

Comment:
This could be a [guard clause](https://medium.com/@scadge/if-statements-design-guard-clauses-might-be-all-you-need-67219a1a981a), so you don't have to nest the entire section.